feeds/public.txt

```diff
@@ -44,7 +44,7 @@ fluid-framework
 @fluidframework/odsp-driver-definitions
 @fluidframework/odsp-driver
 @fluidframework/local-driver
-@fluid-tools/fluidapp-odsp-urlresolver
+@fluidframework/odsp-urlresolver
```

Comment:
Looks like a manual replacement here? This package is already listed on line 43 (and same for the other two package list files). I think we just need to regenerate these by running a full build from the root, and I'd expect this line just disappears.